<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportView</class>
 <widget class="QWidget" name="ReportView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>统计报表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>报表类型:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboReportType">
       <item>
        <property name="text">
         <string>每日汇总</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>周统计</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>月度报表</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>科室统计</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>医生工作量</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>预约统计</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>库存报表</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGenerate">
       <property name="text">
        <string>生成报表</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExportHtml">
       <property name="text">
        <string>导出HTML</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExportCsv">
       <property name="text">
        <string>导出CSV</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>